src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/TasksTest.kt

```diff
@@ -1467,44 +1468,61 @@ class TasksTest : IntegrationTestBase() {
+                repeat(numAssessmentsPending) {
+                  createTask(TaskType.assessment, null, allocatableUser, user, offenderDetails.otherIds.crn)
+                }
 
-                val numAssesmentsPending = 12
-                repeat(numAssesmentsPending) {
-                  `Given an Assessment for Approved Premises`(
-                    allocatedToUser = allocatableUser,
-                    createdByUser = user,
-                    crn = offenderDetails.otherIds.crn,
-                    decision = null,
-                    reallocated = false,
-                  )
+                val numPlacementApplicationsPending = 4
+                repeat(numPlacementApplicationsPending) {
+                  createTask(TaskType.placementApplication, null, allocatableUser, user, offenderDetails.otherIds.crn)
                 }
 
-                val numAssesmentsCompleted7Days = 8
-                repeat(numAssesmentsCompleted7Days) {
+                val numPlacementRequestsPending = 2
+                repeat(numPlacementRequestsPending) {
+                  createTask(TaskType.placementRequest, null, allocatableUser, user, offenderDetails.otherIds.crn)
+                }
+
+                val numAssessmentsCompleted7Days = 4
+                repeat(numAssessmentsCompleted7Days) {
                   val days = kotlin.random.Random.nextInt(1, 7).toLong()
-                  `Given an Assessment for Approved Premises`(
-                    allocatedToUser = allocatableUser,
-                    createdByUser = user,
-                    crn = offenderDetails.otherIds.crn,
-                    decision = null,
-                    reallocated = false,
-                    submittedAt = OffsetDateTime.now().minusDays(days),
-                  )
+                  createTask(TaskType.assessment, OffsetDateTime.now().minusDays(days), allocatableUser, user, offenderDetails.otherIds.crn)
+                }
+
+                val numPlacementApplicationsCompleted7Days = 2
+                repeat(numPlacementApplicationsCompleted7Days) {
+                  val days = kotlin.random.Random.nextInt(1, 7).toLong()
+                  createTask(TaskType.placementApplication, OffsetDateTime.now().minusDays(days), allocatableUser, user, offenderDetails.otherIds.crn)
+                }
+
+                val numPlacementRequestsCompleted7Days = 1
+                repeat(numPlacementRequestsCompleted7Days) {
+                  val days = kotlin.random.Random.nextInt(1, 7).toLong()
+                  createTask(TaskType.placementRequest, OffsetDateTime.now().minusDays(days), allocatableUser, user, offenderDetails.otherIds.crn)
                 }
 
-                val numAssesmentsCompleted30Days = 9
-                repeat(numAssesmentsCompleted30Days) {
+                val numAssessmentsCompleted30Days = 4
+                repeat(numAssessmentsCompleted30Days) {
                   val days = kotlin.random.Random.nextInt(8, 30).toLong()
```

Comment:
i think this should be nextInt(1,30) as we still want to include those in the first 7 days? (this is repeated a few times in this test)

Reply:
If we had that, then we'd potentially have a flickering test scenario, because if they'd been completed between 1 and 7 days ago, they'd show up as being completed in the last 7 days. I'll update some naming to make that clearer, as `numAssessmentsCompleted30Days` suggests they cover ALL that 30 day period